api4/system.go

```diff
@@ -339,7 +340,53 @@ func getLogs(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	lines, appErr := c.App.GetLogs(c.Params.Page, c.Params.LogsPerPage)
+	var logFilter *model.LogFilter
```

Comment:
Maybe, I don't know if it is needed, check here if the configuration of the logs is allowing structured logs (if not, you can't properly parse the log contents).

Reply:
It ultimately doesn't stop the user from turning off structured logging and turning it back on and still using the query endpoint. If they do that it will still fail even if we check that it's not.

0/5 on adding this just as an extra stop-gap.